index.css

```diff
@@ -0,0 +1,63 @@
+body {    
+    background-color: rgb(138, 9, 155);
+    background-size: auto;
```

Comment:
I believe that a lot of these attributes are not necessary. Please review what you really need in the inspector and remove what is not necessary. Unused CSS is a big problem, and in large enough systems it can lead to things breaking everywhere with the slightest change.

Reply:
Done.